ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -97,6 +97,14 @@
     dest: "{{ jenkins.dest }}"
     mode: "{{ jenkins.mode|default('0644') }}"
 
+# TODO - Should this run on every machine?
+- name: copy start.sh to {{ home }}/{{ server_user }}/start.sh
```

Comment:
cc/ @jbergstroem @rvagg @joaocgreis , should we still be putting start.sh on every machine? The new ansible scripts don't so far, but IDK if that's an oversight or a conscious decision.